### PR TITLE
Fixing text column bulleted bug

### LINKED ISSUE #8055: Bulleted text column doesn't show bullets
### Package

filament/tables

### Package Version

v3.0.34

### Laravel Version

v10.20

### Livewire Version

v3

### PHP Version

PHP 8.2

### Problem description

Text column with `->bulleted()` does not show the the bullets because of the `flex` class applied to the `<li>` tag.

### Expected behavior

Showing the bullets.

### Steps to reproduce

Clone the `bug/bulleted-text-column` branch of the reproduction repository
Migrate and seed db
login with test@email.com, password
See Groups table list where text column does not show bullets

### Reproduction repository

https://github.com/ilpav12/filament-bug

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by ilpav12**
- [x] Changes have been thoroughly tested to not break existing functionality.

This PR fixes bug #8055

### CODE REVIEW

File: packages/tables/resources/views/columns/text-column.blade.php [hunk 1]
```diff
@@ -95,7 +96,8 @@
                         "
                     @endif
                     @class([
-                        'flex w-max',
+                        'max-w-max',
```
Comment (zepfietje) [inline packages/tables/resources/views/columns/text-column.blade.php]: This should probably stay `w-max` as it fixes a recent bug.

File: packages/tables/resources/views/columns/text-column.blade.php [hunk 2]
```diff
@@ -11,6 +11,7 @@
     $iconPosition = $getIconPosition();
     $isListWithLineBreaks = $isListWithLineBreaks();
     $url = $getUrl();
+    $isBulleted = $isBulleted();
```
Comment (zepfietje) [inline packages/tables/resources/views/columns/text-column.blade.php]: Could you sort these alphabetically?

**Review by zepfietje — CHANGES_REQUESTED** [review]
Thanks for this PR, @ilpav12.
Could you please provide screenshots though?
The pull request template exists for a reason.

**Comment by ilpav12** [comment]
@zepfietje this is the current behavior:
image
If I manually remove the `flex` class from a `<li>` element the bullet is showing:
image
After the fix it is working correctly:
image

**Comment by zepfietje** [comment]
Thanks!

**Pull request merged**